gateway/src/main/resources/log4j2-spring.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<Configuration status="WARN">
+  <Appenders>
+    <Console name="Console">
+      <PatternLayout pattern="%d{yyyy-MM-dd HH:mm:ss.SSS} %p %notEmpty{[X-Request-ID=%X{X-Request-ID}]} %notEmpty{[trace=%X{traceId}]} %notEmpty{[span=%X{spanId}]} --- [%t] %c : %msg%n" />
+    </Console>
+    <SpringProfile name="jsonlogs">
+      <Console name="Console" target="SYSTEM_OUT" follow="true">
+        <!--  see https://github.com/vy/log4j2-logstash-layout/blob/master/layout/src/main/resources/LogstashJsonEventLayoutV1.json -->
+        <JsonTemplateLayout
+          eventTemplateUri="classpath:LogstashJsonEventLayoutV1.json">
```

Comment:
Any particular reason to use this json template instead of the default one ?

Reply:
I guess the log4j2 needs to specify a one, and LogstashJsonEventLayoutV1.json is one of them. Still, this is one template provided by the library into the log4j-layout-template-json jar:

```
EcsLayout.json
StackTraceElementLayout.json
JsonLayout.json
LogstashJsonEventLayoutV1.json
GcpLayout.json
GelfLayout.json
```

Thinking of where this PR comes from originally, I hardly remember that they were already making use of a JsonEventLayoutV1 from log4j1 in other geOrchestra components, so for me it makes sense if they were configuring the logger onto this template.

We can still try out some other ones and see if they suit better to the future use-case ?

Reply:
I read that EcsLayout is default (if no eventTemplateUri specified). Doesn't look much different. But maybe it's good, still, to have it explcitly configured so we know how it's done.

Reply:
In any case, the accesslog message will still be a CLF string, from what I read, that's what netty does, probably not much to be done on this unless we want to get into some custom complexity